ci/edmtool.py

```diff
@@ -94,13 +89,15 @@
 extra_dependencies = {
     'pyside': {'pyside'},
     'pyqt': {'pyqt'},
+    'pyqt5': {'pyqt5'},
     'wx': {'wxpython'},
     'null': set()
 }
 
 environment_vars = {
     'pyside': {'ETS_TOOLKIT': 'qt4', 'QT_API': 'pyside'},
```

Comment:
we can probably remove this (or maybe we will need to update it if we start testing against `pyside2`)